examples/mallocMC_example01.cu

```diff
@@ -122,10 +122,10 @@ void run()
   std::vector<int> array_sums(block*grid,0);
 
   // create arrays of arrays on the device
-  createArrays<<<1,1>>>(grid,block);
+  createArrays<<<1,1>>>(grid,block, mMC.devAllocator);
```

Comment:
So, would you prefer a getter method (on the host) that returns an object like the following? (should be passed to the kernel as a pass-by-value object)

template <typename T_DevAllocator>
struct AllocatorHandle
{
    T_DevAllocator* devAllocator;

    MAMC_ACCELERATOR
    void*
    malloc(
        size_t size
    )
    {
        return devAllocator -> malloc( size );
    }

    MAMC_ACCELERATOR
    void
    free(
        void* p
    )
    {
        devAllocator -> free( p );
    }

};